config/e3sm/config_grids.xml

```diff
@@ -168,7 +168,7 @@
       <mask>360x720cru</mask>
     </model_grid>
 
-    <model_grid alias="NLDAS_NLDAS" compset="(DATM_SATM).+CLM">
```

Comment:
OK, I don't get this change. And I don't think it's correct either. We have compsets that run NLDAS with either MOSART or SROF, and need it to function either way. Do you need to enable compsets that aren't running CLM, but are running MOSART?

Reply:
@jonbob ^

Reply:
I was asked to support a configuration in which MOSART is the only active model, so yes.

Reply:
Yes this "MOSART only" compset was originally added by @apcraig in a old E3SM development branch and was recently merged to E3SM master by @jonbob . River model developers like myself need this configuration to run MOSART without running the land model (to save time).

Reply:
I think @ekluzek 's point is that you shouldn't need to explicitly specify MOSART here. You can just delete the "CLM" from the compset match without replacing it with anything else. Actually, I'm guessing that the important thing here is that you don't have an active atmosphere and don't have an active ocean, or something like that.

But I won't hold this PR up for this particular point, since at least this change doesn't impact models other than E3SM.